service/watcher/cache.go

```diff
@@ -0,0 +1,289 @@
+package watcher
```

Comment:
This isn't a new file rather a renaming, not so sure why it completely see it as a new file.